### DISCUSSION TITLE
Disable zoom on scroll when scrolling inside nodes

### DISCUSSION THREAD

**Discussion by omar-shahid** [xyflow/xyflow#2372] [Q&A] [answered]
Is there a way to not register zoom on scroll when the cursor is inside the node? Currently, whenever the user scrolls inside the node (assuming it will scroll the node instead of zooming in), the react flow doesn't seem to register it. Is there a way or an option in the react flow?
![image](https://user-images.githubusercontent.com/40920737/185569997-ceee164d-365a-492b-bd8b-b5c6ef1885fe.png)

Here is my react flow config: 
![image](https://user-images.githubusercontent.com/40920737/185570239-8568ad9b-f9e0-4f48-83b4-599d5f6ab7e4.png)

**Comment by bcakmakoglu** [comment, marked as answer]
You can allow scrolling inside nodes by adding the `nowheel` classname to your nodes.
[Here's the relevant docs.](https://reactflow.dev/docs/api/nodes/custom-nodes/#allow-scrolling-inside-a-node)

**Reply by moklick** [reply]
yes! thanks @bcakmakoglu

**Reply by omar-shahid** [reply]
Thank you very much! How can we use the default cursors inside the node? Currently it shows a hand, even though I have disabled dragging as you can see above.

**Reply by omar-shahid** [reply]
@moklick @bcakmakoglu should I open new discussion for this question?

**Reply by moklick** [reply]
https://user-images.githubusercontent.com/2857535/185719115-2dc3f189-c77e-4ef5-86fb-90ea86ba62c6.MOV

This is the new default behavior in v11. We will release it in the next weeks
